nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -185,33 +188,25 @@ let
       # So instead we'll list a lot of possibly valid modes :/
       #"3840x2160"
       #"2560x1440"
+      "1920x1200"
       "1920x1080"
       "1366x768"
+      "1280x800"
       "1280x720"
+      "1200x1920"
       "1024x768"
+      "800x1280"
       "800x600"
       "auto"
     ]}
 
-    # Fonts can be loaded?
-    # (This font is assumed to always be provided as a fallback by NixOS)
-    if loadfont (\$root)/EFI/boot/unicode.pf2; then
```

Comment:
is font loading getting removed because loading fonts breaks something, or because its useless, or because `with_fonts` check is somehow broken?

Reply:
It's not needed anymore, as this was for the "gfxterm"-based menu, and instead we rely on the UEFI intrinsic console output.